runners/gradle-plugin/src/test/kotlin/org/jetbrains/dokka/gradle/GradleDokkaSourceSetBuilderTest.kt

```diff
@@ -113,7 +113,7 @@ class GradleDokkaSourceSetBuilderTest {
         sourceSet.dependsOn(sourceSet.DokkaSourceSetID("s3"))
         sourceSet.dependsOn(GradleDokkaSourceSetBuilder("s4", project))
         sourceSet.dependsOn(GradleDokkaSourceSetBuilder("s5", project).build())
-        sourceSet.dependsOn(DefaultKotlinSourceSet(project, "s6"))
```

Comment:
Looks like there was some refactoring of gradle api, `DefaultKotlinSourceSet` is now abstract, so cannot instantiate it directly.